feat/new product overview pages (#646)
hirosystems/docs: docs/hiro-archive/index.mdx

    View the Hiro Archive <span classNam="ml-1 no-underline">→</span>
  </a>
</div>

### Supported Services

We record and archive mainnet and testnet snapshots for the following services every night:

<div class="subSections my-8">
  <a href="/hiro-archive/guides/use-stacks-blockchain-archive">
    <div class="subSectionTitle"><h3>Stacks Blockchain</h3><span>→</span></div>
    <p>Quickly spin up a new Stacks node.</p>
  </a>
  <a href="/hiro-archive/guides/use-stacks-blockchain-api-archive">
    <div class="subSectionTitle"><h3>Stacks Blockchain API</h3><span>→</span></div>
    <p>Spin up a new instance of the Stacks Blockchain API.</p>
  </a>
</div>

<div class="subSections my-8">
  <a href="/hiro-archive/guides/use-token-metadata-api-archive">
    <div class="subSectionTitle"><h3>Token Metadata API</h3><span>→</span></div>
    <p>Spin up a new instance of the Token Metadata API.</p>
  </a>
  <a href="/hiro-archive/guides/verify-archive-data">
    <div class="subSectionTitle"><h3>Verify archive data</h3><span>→</span></div>
    <p>Verify the accuracy of data recorded in a particular snapshot.</p>
  </a>
</div>

---

> Need help building with the Hiro Archive? Reach out to us on the `#api` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with the Hiro Archive? Reach out to us on the `#api` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
